batchflow/monitor.py

```diff
@@ -215,12 +271,14 @@ def get_usage(gpu_list=None, **kwargs):
     USSMonitor: ['uss'],
     GPUMonitor: ['gpu'],
     GPUMemoryMonitor: ['gpu_memory'],
+    GPUMemoryUtilizationMonitor: ['gpu_memory_utilization']
 }
 
 MONITOR_ALIASES = {alias: monitor for monitor, aliases in MONITOR_ALIASES.items()
                    for alias in aliases}
 
 
+
 @contextmanager
 def monitor_resource(resource='memory', frequency=0.5, **kwargs):
```

Comment:
```suggestion
def resource_monitor(resource='memory', frequency=0.5, **kwargs):
```

Reply:
The way it is now, it is an action: `while monitoring those resources, do...:`